managed/services/management/rds.go

```diff
@@ -180,6 +180,13 @@ func (s *RDSService) DiscoverRDS(ctx context.Context, req *managementpb.Discover
 			for _, db := range regInstances {
 				l.Debugf("Discovered instance: %+v", db)
 
+				// This happens when the database is in "Creating" state.
+				// At this point there is no endpoint available.
+				if db.Endpoint == nil {
+					l.Infof("Instance %q not ready yet. Please wait until the database is fully created in AWS.", *db.DBInstanceIdentifier)
```

Comment:
let's print it at debug level

Reply:
Sure, no problem. Originally I thought this would be helpful to admins to figure out why they don't see the instances they expect, but debug is fine as well.